lib/heroku/command/domains.rb

```diff
@@ -13,17 +14,35 @@ class Domains < Base
     #Examples:
     #
     # $ heroku domains
-    # === Domain names for example
-    # example.com
+    # === Development Domain
+    # example.herokuapp.com
+    #
+    # === Custom Domains
+    # Domain Name  CNAME Target
```

Comment:
For the UI, we were looking for a name that implied it was the "right hand
side" of the CNAME record, e.g., the canonical name, not one of several
nicknames/aliases/etc.

Don't think we should block either but let me know if there are strong
opinions or I am misunderstanding. :)

On Wed, May 27, 2015 at 10:48 PM, Ryan Brainard <notifications@github.com>
wrote:

> In lib/heroku/command/domains.rb
> <https://github.com/heroku/heroku/pull/1589#discussion_r31204800>:
>
> > @@ -13,17 +14,35 @@ class Domains < Base
> >      #Examples:
> >      #
> >      # $ heroku domains
> > -    # === Domain names for example
> > -    # example.com
> > +    # === Development Domain
> > +    # example.herokuapp.com
> > +    #
> > +    # === Custom Domains
> > +    # Domain Name  CNAME Target
>
> there was a lot of discussion about the wording for this column in
> Dashboard, and this was copied from there. @tim-lang
> <https://github.com/tim-lang>, @itchymutt <https://github.com/itchymutt>:
> thoughts? I don't think we should block on this, but worth consideration
> for a follow up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/heroku/heroku/pull/1589/files#r31204800>.
>